src/utils/misc.py

```diff
@@ -41,18 +41,22 @@ def match(self, hwnd) -> bool:
             result = False
         return result
 
+specs = [
+    WindowSpec(title_regex="D2R1 on .+"),
+    WindowSpec(process_name_regex="D2R.exe")
+]
 
```

Comment:
@Ezro Thanks for rebasing. I see you've adapted the changes from #563 but I don't see the new params used anywhere, which I think was the purpose of that PR to allow customization. Would you see if you can incorporate these? The new params were appropriately merged under [advanced_options] in params.ini but don't look like they got put into Config in the rebase and aren't used here. Thanks

Reply:
This should be resolved now. I tried tweaking the implementation during the rebase, but I've reverted it back to how it is in main.